src/odyssey.ts

```diff
@@ -0,0 +1,93 @@
+/**
+ * Copyright (c) AcademyJS and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { Command } from "commander";
+import chalk from "chalk";
+import mongoose from "mongoose";
+import { readFileSync } from "fs";
+import dotenv from "dotenv";
+import { SyncConfiguration } from "./types";
+import { TestSuite } from "./models";
+
+dotenv.config();
+const { DATABASE_URL } = process.env;
+
+const syncExercises = async (
+    configuration: SyncConfiguration
+): Promise<void> => {
+    const file = configuration.file ?? "meta.json";
+    const rawData = readFileSync(file);

```

Comment:
You're still using `readFileSync`. :/

Anyway, this should do. Personally, I'd go with what I suggested.